pkgs/development/python-modules/networkx/default.nix

```diff
@@ -7,12 +7,13 @@
 
 buildPythonPackage rec {
   pname = "networkx";
-  version = "2.1";
+  # upgrade may break sage, please test the sage build or ping @timokau on upgrade
```

Comment:
there are a lot of packages that can break sage so I don't see much value in this comment

Reply:
There are not *that* many that regularly break sage. And even if there are: I have to fix it after the upgrade anyways. Easier for everybody if it is tested and/or I get notified before the breakage happens.